assets/style/services/services.scss

```
@@ -5,3 +5,329 @@
 @import "../globalStyle/footer";
 
 // Author: Olga
+
+.global-container {
+  max-width: 1920px;
+  width: 100%;
+  margin: 0 auto;
+}
+.main__services {
+  font-family: $font;
+  min-height: 420px;
```

Не стоит задавать параметр min-height для контейнеров. В случае, если контент внутри контейнера уменьшится в ходе работы какого-то скрипта, у нас появится 420px пустого места, что как бы не очень будет хорошо смотреться. Я бы удалил min-height и вместо него ориентировался на высоту самого контента и на padding-block
```suggestion
  padding-block: 107px 163px;
```